src/backend/u64/field.rs

```diff
@@ -524,6 +524,7 @@ impl FieldElement {
     /// Load a `FieldElement` from the low 253b bits of a 256-bit
     /// input. So Little Endian representation in bytes of a FieldElement.
     // @TODO: Macro for Inline load8 function as it has variadic arguments.
+    #[warn(dead_code)]
```

Comment:
Why is this added? from_bytes() is a function called in the lib already, so it shouldn't need to use the warning disablement.

Reply:
As the ultimate goal is for complete Ristretto injection, I figured showing how the curve code can work using this could be of importance. To look at factors such as the prime field as a whole and give a justification of why they are set (in our case, ristretto), then insight into the functions can be useful.